src/molecules/card-chip-main/index.jsx

```diff
@@ -0,0 +1,62 @@
+import {
+	Box,
+	CardActionArea,
+	CardActions,
+	CardContent,
+	CardMedia,
+	IconButton,
+	Stack,
+	Tooltip,
+	Typography,
+} from "@mui/material";
+import { ExpandMore, LocationOn, Message } from "@mui/icons-material";
+import React from "react";
+import useStore from "/src/ions/store/index.jsx";
+
+const CardChipMain = ({ card, index }) => {
+	const setExpanded = useStore(state => state.setExpanded);
+	const cards = useStore(state => state.cards);
+
+	return (
+		<Stack direction="row">
+			<CardActionArea style={{ position: "relative" }}>
+				<CardMedia
+					component="img"
+					image={card.image}
+					alt={`Kite-Instructor ${card.name}`}
+				/>
+			</CardActionArea>
+			<Box width={400}>
+				<CardContent>
+					<Typography variant="h5">{card.name}</Typography>
+					<Typography variant="body1">
+						<LocationOn />
+						{card.location}
+						<br />
+						50€ per hour
+					</Typography>
+				</CardContent>
+				<CardActions>
+					<Tooltip title="Expand">
+						<IconButton
+							aria-expanded={cards[index].isExpanded}
+							aria-label="show more"
+							onClick={() => {
+								setExpanded(index);
+							}}
+						>
+							<ExpandMore />
```

Comment:
You could rotate the icon or use a different icon when expanded. 

Right now, the Icon does not indicate if the section is expanded or collapsed.

Reply:
so true, is set to styling ticket to be fixed